packages/ra-ui-materialui/src/form/FormTab.tsx

```diff
@@ -53,6 +52,12 @@ export const FormTab = (props: FormTabProps) => {
                 // notes that aria-hidden="false" currently behaves inconsistently across browsers.
                 aria-hidden={hidden || undefined}
                 {...rest}
+                sx={[
+                    {
+                        alignItems: 'flex-start',
+                    },
+                    ...(Array.isArray(rest.sx) ? rest.sx : [rest.sx]),
+                ]}
```

Comment:
I don't get this: why not just:

```suggestion
                sx={{
                        alignItems: 'flex-start',
                        ...rest.sx,
                }}
```

Reply:
Hi @djhi,

These are auto applied by the codemod tool in https://mui.com/material-ui/migration/migrating-from-deprecated-apis/#system-props.

Main reason we did not directly unpack ...rest.sx, but we converted to an array is because the typing of SxProp can also be functions. MUI suggests using array values. https://mui.com/system/getting-started/the-sx-prop/#array-values